src/components/PageWrapper/style.ts

```diff
@@ -19,11 +15,11 @@ export const Content = styled.div`
   > div {
     flex: 1;
   }
-`
+`;
 
 export const Footer = styled.div`
   text-align: center;
   color: white;
-  background: ${darkblue};
+  background: #343a40;
```

Comment:
Er det ikke greit å fortsette å bruke variabeler for farger?

Reply:
Yess, please do `(Y)`